### PR TITLE
change attribute sort="ascending" for live-list feature

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: extensions/amp-live-list/0.1/amp-live-list.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ export class AmpLiveList extends AMP.BaseElement {
         actualCount);
 
     if (isExperimentOn(this.win, 'amp-live-list-sorting')) {
-      this.isReverseOrder_ = this.element.hasAttribute('reverse');
+      this.isReverseOrder_ = this.element.gettAttribute('sort') === 'ascending';
```
Comment (ericlindley-g) [inline extensions/amp-live-list/0.1/amp-live-list.js]: typo (I think?): gettAttribute -> getAttribute

**Comment by ericlindley-g** [comment]
Attribute name change LGTM — thanks!

File: extensions/amp-live-list/0.1/amp-live-list.js (hunk 1 above)
Comment (erwinmombay) [reply extensions/amp-live-list/0.1/amp-live-list.js]: @ericlindley-g good catch! LOL

Comment (ericlindley-g) [reply extensions/amp-live-list/0.1/amp-live-list.js]: Wishing I had a "Hackerman" GIF with my face photoshopped in that I could put here...

**Comment by erwinmombay** [comment]
force merged. visual diffs were timing out and need this to go into canary

**Pull request merged**